moz_kinto_publisher/main.py

```diff
@@ -594,18 +594,6 @@ def publish_intermediates(*, args, rw_client, new_filter_run_id=None):
         except KeyError:  # raw_record doesn't have "id"
             log.error(f"Couldn't delete record: {raw_record}")
 
-    # Delete any expired remote records
-    for unique_id in remote_expired:
```

Comment:
So is `remote_expired` used any longer?

Reply:
Yes, we log the number of expired certs on line 565. I thought about deleting it, but it's nice to have an explanation for some removals.

Reply:
Okay, sounds good.